test_runner/fixtures/neon_fixtures.py

```diff
@@ -127,6 +127,22 @@
 DEFAULT_BRANCH_NAME: str = "main"
 
 BASE_PORT: int = 15000
+# Combination: 0: old, 1: new
```

Comment:
Could you add more verbose comment here, please

Reply:
This code was removed